src/main/java/pl/tomaszdziurko/jvm_bloggers/view/panels/CustomPageLinkModel.java

```diff
@@ -0,0 +1,51 @@
+package pl.tomaszdziurko.jvm_bloggers.view.panels;
+
+import lombok.RequiredArgsConstructor;
+
+import org.apache.wicket.markup.html.navigation.paging.IPageable;
+import org.apache.wicket.model.IModel;
+
+import java.io.Serializable;
+
+@RequiredArgsConstructor
+public class CustomPageLinkModel implements IModel<String>, Serializable {
+
+    protected final IPageable pageable;
+    protected final long pageNumber;
+    private final String css;
+
+    @Override
+    public String getObject() {
+        return isEnabled() ? css : "";
+    }
+
+    @Override
+    public void setObject(String object) {
+    }
+
+    @Override
+    public void detach() {
+    }
+
```

Comment:
Empty `detach()` and `setObject()` methods can be removed if you extend  [`AbstractReadOnlyModel`](https://ci.apache.org/projects/wicket/apidocs/6.x/org/apache/wicket/model/AbstractReadOnlyModel.html) instead implementing `IModel` directly.